src/cobalt.rs

```diff
@@ -115,44 +113,47 @@ pub fn build(config: &Config) -> Result<()> {
     // January 1, 1970 0:00:00 UTC, the beginning of time
     let default_date = UTC.timestamp(0, 0).with_timezone(&FixedOffset::east(0));
 
+    let (mut posts, documents): (Vec<Document>, Vec<Document>) = documents.into_iter()
+        .partition(|x| x.is_post);
+
     // sort documents by date, if there's no date (none was provided or it couldn't be read) then
     // fall back to the default date
-    documents.sort_by(|a, b| b.date.unwrap_or(default_date).cmp(&a.date.unwrap_or(default_date)));
+    posts.sort_by(|a, b| b.date.unwrap_or(default_date).cmp(&a.date.unwrap_or(default_date)));
 
     // check if we should create an RSS file and create it!
     if let &Some(ref path) = &config.rss {
-        try!(create_rss(path, dest, &config, &documents));
+        try!(create_rss(path, dest, &config, &posts));
     }
 
     // these are the attributes of all documents that are posts, so that they can be
     // passed to the renderer
-    // TODO: do we have to clone these?
-    let post_data: Vec<Value> = documents.iter()
-        .filter(|x| x.is_post)
+    let mut post_data: Vec<Value> = posts.iter()
```

Comment:
I understand the naming comes from previous revisions... however

name `post_data` is a little bit confusing. i'd call this at least `posts_data`